server/graphql/loaders/index.js

```diff
@@ -159,6 +159,36 @@ export const loaders = req => {
         })
         .then(results => sortResults(ids, results, 'CollectiveId')),
     ),
+    activeRecurringContributions: new DataLoader(ids =>
+      models.Order.findAll({
+        attributes: [
+          'CollectiveId',
+          'interval',
+          [sequelize.fn('COALESCE', sequelize.fn('SUM', sequelize.col('totalAmount')), 0), 'total'],
+        ],
+        where: {
+          [Op.and]: [{ CollectiveId: { [Op.in]: ids } }, { interval: { [Op.not]: null } }],
```

Comment:
We need to make sure that order has `ACTIVE` `status` to not include cancelled subscriptions